transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -378,10 +380,11 @@ protected void finishModelStage(final OperationContext context, final ModelNode
                     ModelNode resourceModel = model.getModel();
                     if (resourceModel.hasDefined(mutualAttributeName) && resourceModel.get(mutualAttributeName).asBoolean()) {
                         resourceModel.get(mutualAttributeName).set(new ModelNode(false));
+                        context.addResponseWarning(Level.WARNING, "Please note that only journal store or jdbc store can be used, thus '" + mutualAttributeName + "' has been disabled.");
```

Comment:
@tomjenkinson  yes it does. Also @claudio4j mentioned I should be using the "alternatives" metadata, rather than using a simple warning. I'm going to look into that, and update this PR accordingly. Thanks for you all the feedback !